src/views/v2/components/loader/loader.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * View: Loader
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/components/loader/loader.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @version TBD
+ */
+
+if ( empty( $loader_classes ) ) {
+	$loader_classes = $this->get( 'classes' ) ?: [];
+}
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: 832a1e71d202afd7b14aefa58c71f7d3</sub>